<?xml version="1.0"?>

<launch>
    <arg name="bag" default="$(find rpz_planning)/data/josef_2019-06-06-13-58-12_proc_0.1m.bag"/>
    <arg name="bag_rate" default="10.0"/>
    <arg name="play_bag" default="false"/>
    <arg name="rviz" default="false"/>

    <param name="use_sim_time" value="true"/>

    <node name="points_occlusion" pkg="rpz_planning" type="points_occlusion.py" output="screen">
        <param name="pointcloud_topic" value="/final_cost_cloud"/>
        <param name="min_dist" value="1.0"/>
        <param name="max_dist" value="10.0"/>
    </node>

    <node if="$(arg play_bag)" name="play" pkg="rosbag" type="play" output="screen"
          args="-r $(arg bag_rate) -k $(arg bag)"/>

    <node if="$(arg rviz)" name="rviz" pkg="rviz" type="rviz"
          args="-d $(dirname)/../rviz/occlusion.rviz"/>
</launch>
